File: app/src/main/java/com/fidoo/user/data/model/Product.java
package com.fidoo.user.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Product {
    @SerializedName("product_id")
    @Expose
    public String productId;
    @SerializedName("quantity")
    @Expose
    public String quantity;
    @SerializedName("is_customize")
    @Expose
    public String isCustomize;
    @SerializedName("message")
    @Expose
    public String message;
    @SerializedName("customize_sub_cat_id")
    @Expose
    public List<String> customizeSubCatId = null;

    public Product(String productId,List<String> customizeSubCatId, String isCustomize, String message, String quantity) {
        this.productId = productId;
        this.customizeSubCatId = customizeSubCatId;
        this.isCustomize = isCustomize;
        this.message = message;
        this.quantity = quantity;
    }

    public Product() {

    }
}

File: app/src/main/java/com/fidoo/user/newRestaurants/model/Product.kt
package com.fidoo.user.newRestaurants.model

data class Product(
    val cart_id: String,
    val cart_quantity: Int,
    val company_name: String,
    val contains_egg: String,
    val coupon_applied: String,
    val customize_item: List<CustomizeItem>,
    val discount: Int,
    val image: String,
    val in_out_of_stock_status: String,
    val is_customize: String,
    val is_customize_quantity: Int,
    val is_nonveg: String,
    val is_prescription: String,
    val offer_price: String,
    val price: String,
    val product_desc: String,
    val product_id: String,
    val product_name: String,
    val product_sub_category_id: String,
    val unit: String,
    val weight: String
)